src/rules/media-feature-name-blacklist/__tests__/index.js

```diff
@@ -0,0 +1,60 @@
+﻿import {
+  messages,
+  ruleName,
+  } from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [[
+    "max-width",
+    "--max-width",
+    "width",
+    "/^my-/",
+  ]],
+
+  accept: [ {
+    code: "@media (min-width: 50em) { padding: 1rem; }",
+  }, {
+    code: "@media (min-width: 50em) { max-width: 50em; }",
+  }, {
+    code: "@media (--max-width: 50em) { padding: 1rem; }",
+    description: "ignore custom media query",
+  }, {
+    code: "@media (/* max-width: 50em */ min-width: 50em) { padding: 1rem; }",
+    description: "ignore comments",
+  }, {
+    code: "@media (width <= 50em) { padding: 1rem; }",
+    description: "ignore media features in a range context",
+  } ],
+
+  reject: [ {
+    code: "@media (max-width: 50em) { padding: 1rem; }",
+    message: messages.rejected("max-width"),
+    line: 1,
+    column: 9,
+  }, {
+    code: "@media (MaX-wIdTh: 50em) { padding: 1rem; }",
+    message: messages.rejected("MaX-wIdTh"),
+    line: 1,
+    column: 9,
+  }, {
+    code: "@media print and (max-width: 50em) { padding: 1rem; }",
+    message: messages.rejected("max-width"),
+    line: 1,
+    column: 19,
+  }, {
+    code: "@media handheld and (min-width: 20em), screen and (max-width: 20em) { padding: 1rem; }",
+    message: messages.rejected("max-width"),
+    line: 1,
+    column: 52,
+  }, {
+    code: "@media (my-width: 50em) { padding: 1rem; }",
+    message: messages.rejected("my-width"),
+    line: 1,
+    column: 9,
+  } ],
+})
```

Comment:
Can we make sure that media feature names containing SCSS & Less variables and operators are ignored?